README.md

```diff
@@ -1,2 +1,200 @@
+<img align="right" src="https://github.com/moderepo/sensor-cloud-eval/blob/master/sceval_frontend/src/common_images/mode-logo.png">
+
 # Sensor-Cloud-Eval
-A TSX version of Sensor Cloud to be used as an evaluation kit by partners.
+
+Sensor Cloud Eval is a **TSX version** of MODE's Sensor Cloud, developed to be used as an evaluation kit by business partners and app developers.
+
+## Contents
+
+- [Features](#features)
+- [Installation and Setup](#installation-and-setup)
+- [Prerequisites](#prerequisites)
+- [Quick start](#quick-start)
+- [Mode API Reference](https://dev.tinkermode.com/docs/api/)
+- [API Examples](#api-examples)
+  - [Using REST Endpoints (GET, POST, PUT, PATCH, DELETE)](#using-rest-endpoints)
+    - [Handling User Information](#handling-user-information)
+    - [Linking / Unlinking Sensor Modules](#linking-and-unlinking-sensor-modules)
+    - [Sending Device Commands](#sending-device-commands)
+  - [Receiving Device Events (Websocket)](#receiving-device-events)
+- [Links](#links)
```

Comment:
@mattthewong, should we add another section to explain the UI, especially the Hardware page and Sensor Module page?
**Hardware page** - We don't have a way to add Home or devices in this app so maybe need to let them know they need to use the App Simulator to add a home and use Device Simulator to add devices to a home. I was confused too at first and wondered where these devices (Gateway 34114 and 34165) came from.
**Add Sensor Module page** - Maybe explain a little bit about how this works.
**Sensor Module** - I don't know if the devs already know about how we use key/value store for configuring/linking sensor modules from reading other documentation but if not, it might be helpful to explain this. Sensor modules are not really a thing in Mode platform. It is kind of something we created for the Gateway using key/value store system. This part was confusing for me too at first and I figured it out by going through the code and tracing all the API calls.

Reply:
Sure, let me add these sections. I mentioned this briefly in the `prerequisites`, but I think it may be better to make this fully explicit. Let me adjust this shortly.